<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>JaabaPluginDialog</class>
 <widget class="QDialog" name="JaabaPlugin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>777</width>
    <height>331</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Jaaba Plugin</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QWidget" name="statusWidgetPtr" native="true">
     <property name="font">
      <font>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QTabWidget" name="tabWidgetPtr">
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="ParamsTabPtr">
         <attribute name="title">
          <string>Params</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_6">
          <item>
           <widget class="QGroupBox" name="comGroupBoxPtr">
            <property name="font">
             <font>
              <weight>50</weight>
              <bold>false</bold>
             </font>
            </property>
            <property name="title">
             <string>Views</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_5">
             <item>
              <widget class="QWidget" name="commWidgetPtr" native="true">
               <layout class="QHBoxLayout" name="horizontalLayout_10">
                <item>
                 <widget class="QCheckBox" name="sideRadioButtonPtr_">
                  <property name="text">
                   <string>side</string>
                  </property>
                  <property name="checked">
                   <bool>false</bool>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QCheckBox" name="frontRadioButtonPtr_">
                  <property name="text">
                   <string>front</string>
                  </property>
                  <property name="checked">
                   <bool>false</bool>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer_4">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>10</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QGroupBox" name="comGroupBoxPtr_1">
            <property name="font">
             <font>
              <weight>50</weight>
              <bold>false</bold>
             </font>
            </property>
            <property name="title">
             <string>HOGHOF Param file locations</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout">
             <item>
              <widget class="QWidget" name="commWidgetPtr_1" native="true">
               <layout class="QVBoxLayout" name="verticalLayout_1">
                <item>
                 <widget class="QWidget" name="commWidgetPtr_2" native="true">
                  <layout class="QHBoxLayout" name="horizontalLayout_1">
                   <item>
                    <widget class="QLabel" name="comPortLabelPtr1">
                     <property name="sizePolicy">
                      <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
                       <horstretch>0</horstretch>
                       <verstretch>0</verstretch>
                      </sizepolicy>
                     </property>
                     <property name="minimumSize">
                      <size>
                       <width>80</width>
                       <height>0</height>
                      </size>
                     </property>
                     <property name="maximumSize">
                      <size>
                       <width>150</width>
                       <height>16777215</height>
                      </size>
                     </property>
                     <property name="font">
                      <font>
                       <weight>50</weight>
                       <bold>false</bold>
                      </font>
                     </property>
                     <property name="text">
                      <string>Path to directory</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QLineEdit" name="pathtodir_">
                     <property name="placeholderText">
                      <string></string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QPushButton" name="reloadPushButtonPtr_">
                     <property name="font">
                      <font>
                       <weight>50</weight>
                       <bold>false</bold>
                      </font>
                     </property>
                    <property name="text">
                     <string>Reload</string>
                    </property>
                    <property name="autoDefault">
                     <bool>false</bool>
                    </property>
                   </widget>
                  </item>
                  </layout>
                 </widget>
                </item>
                <item>
                 <widget class="QWidget" name="commWidgetPtr_3" native="true">
                  <layout class="QHBoxLayout" name="horizontalLayout_2">
                   <item>
                    <widget class="QLabel" name="comPortLabelPtr2">
                     <property name="sizePolicy">
                      <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
                       <horstretch>0</horstretch>
                       <verstretch>0</verstretch>
                      </sizepolicy>
                     </property>
                     <property name="minimumSize">
                      <size>
                       <width>80</width>
                       <height>0</height>
                      </size>
                     </property>
                     <property name="maximumSize">
                      <size>
                       <width>150</width>
                       <height>16777215</height>
                      </size>
                     </property>
                     <property name="font">
                      <font>
                       <weight>50</weight>
                       <bold>false</bold>
                      </font>
                     </property>
                     <property name="text">
                      <string>HOG File Location</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QLineEdit" name="HOGParamFilePtr_">
                     <property name="placeholderText">
                      <string>json_files/HOGparam.json</string>
                     </property>                 
                    </widget>
                   </item>
                  </layout>
                 </widget> 
                </item>
                <item>
                 <widget class="QWidget" name="commWidgetPtr_4" native="true">
                  <layout class="QHBoxLayout" name="horizontalLayout_4">
                   <item>
                    <widget class="QLabel" name="comPortLabelPtr3">
                     <property name="sizePolicy">
                      <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
                       <horstretch>0</horstretch>
                       <verstretch>0</verstretch>
                      </sizepolicy>
                     </property>
                     <property name="minimumSize">
                      <size>
                       <width>80</width>
                       <height>0</height>
                      </size>
                     </property>
                     <property name="maximumSize">
                      <size>
                       <width>150</width>
                       <height>16777215</height>
                      </size>
                     </property>
                     <property name="font">
                      <font>
                       <weight>50</weight>
                       <bold>false</bold>
                      </font>
                     </property>
                     <property name="text">
                      <string>HOF File Location</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QLineEdit" name="HOFParamFilePtr_">
                     <property name="placeholderText">
                      <string>json_files/HOFparam.json</string>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="CropcomGroupBoxPtr">
            <property name="font">
             <font>
              <weight>50</weight>
              <bold>false</bold>
             </font>
            </property>
            <property name="title">
             <string>Crop Param file locations</string>
            </property>
            <layout class="QVBoxLayout" name="CropverticalLayout_1">
             <item>
              <widget class="QWidget" name="CropcommWidgetPtr_1" native="true">
               <layout class="QVBoxLayout" name="CropverticalLayout_2">
                <item>
                 <widget class="QWidget" name="CropcommWidgetPtr_2" native="true">
                  <layout class="QHBoxLayout" name="CrophorizontalLayout_1">
                   <item>
                    <widget class="QLabel" name="CropcomPortLabelPtr1">
                     <property name="sizePolicy">
                      <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
                       <horstretch>0</horstretch>
                       <verstretch>0</verstretch>
                      </sizepolicy>
                     </property>
                     <property name="minimumSize">
                      <size>
                       <width>80</width>
                       <height>0</height>
                      </size>
                     </property>
                     <property name="maximumSize">
                      <size>
                       <width>250</width>
                       <height>16777215</height>
                      </size>
                     </property>
                     <property name="font">
                      <font>
                       <weight>50</weight>
                       <bold>false</bold>
                      </font>
                     </property>
                     <property name="text">
                      <string>Crop Side File Location</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QLineEdit" name="CropSideParamFilePtr_">
                     <property name="placeholderText">
                      <string>json_files/Cropsde_param.json</string>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </widget>
                </item>
                <item>
                 <widget class="QWidget" name="CropcommWidgetPtr_3" native="true">
                  <layout class="QHBoxLayout" name="CrophorizontalLayout_2">
                   <item>
                    <widget class="QLabel" name="CropcomPortLabelPtr2">
                     <property name="sizePolicy">
                      <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
                       <horstretch>0</horstretch>
                       <verstretch>0</verstretch>
                      </sizepolicy>
                     </property>
                     <property name="minimumSize">
                      <size>
                       <width>80</width>
                       <height>0</height>
                      </size>
                     </property>
                     <property name="maximumSize">
                      <size>
                       <width>250</width>
                       <height>16777215</height>
                      </size>
                     </property>
                     <property name="font">
                      <font>
                       <weight>50</weight>
                       <bold>false</bold>
                      </font>
                     </property>
                     <property name="text">
                      <string>Crop Front File Location</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QLineEdit" name="CropFrontParamFilePtr_">
                     <property name="placeholderText">
                      <string>json_files/Cropfrt_param.json</string>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="ClasscomGroupBoxPtr">
            <property name="font">
             <font>
              <weight>50</weight>
              <bold>false</bold>
             </font>
            </property>
            <property name="title">
             <string>Classifier file locations</string>
            </property>
            <layout class="QVBoxLayout" name="ClassverticalLayout_1">
             <item>
              <widget class="QWidget" name="ClasscommWidgetPtr_1" native="true">
               <layout class="QVBoxLayout" name="ClassverticalLayout_2">
                <item>
                 <widget class="QWidget" name="ClasscommWidgetPtr_2" native="true">
                  <layout class="QHBoxLayout" name="ClasshorizontalLayout_1">
                   <item>
                    <widget class="QLabel" name="ClasscomPortLabelPtr1">
                     <property name="sizePolicy">
                      <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
                       <horstretch>0</horstretch>
                       <verstretch>0</verstretch>
                      </sizepolicy>
                     </property>
                     <property name="minimumSize">
                      <size>
                       <width>80</width>
                       <height>0</height>
                      </size>
                     </property>
                     <property name="maximumSize">
                      <size>
                       <width>250</width>
                       <height>16777215</height>
                      </size>
                     </property>
                     <property name="font">
                      <font>
                       <weight>50</weight>
                       <bold>false</bold>
                      </font>
                     </property>
                     <property name="text">
                      <string> Classifier File Location</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QLineEdit" name="ClassFilePtr_">
                     <property name="placeholderText">
                      <string>json_files/multiclassifier.mat</string>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </widget>
                </item>
                </layout>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="detectButtonPtr_">
            <property name="text">
             <string>Detect</string>
            </property>
            <property name="maximumSize">
              <size>
               <width>100</width>
               <height>100</height>
              </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="saveButtonPtr_">
            <property name="text">
             <string>Save</string>
            </property>
            <property name="maximumSize">
              <size>
               <width>100</width>
               <height>100</height>
              </size>
            </property>
           </widget>
          </item>
         </layout> 
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item> 
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
